packages/verkle/src/types.ts

```diff
@@ -8,6 +8,7 @@ import type { VerkleCrypto as VerkleFFI } from 'verkle-cryptography-wasm'
 // Field representation of a commitment
 export interface Fr {}
 
+// TODO: Decide if we still need this interface.  Commitments are now all bytes from the JS perspective
```

Comment:
I think we can do without this. We are not interacting with elliptic curve point representations directly, and always interfacing through verklecrypto